traffic_portal/app/src/common/modules/table/deliveryServices/TableDeliveryServicesController.js

```diff
@@ -337,13 +337,9 @@ var TableDeliveryServicesController = function(deliveryServices, $anchorScroll,
         },
         null, // Divider
         {
-            text: 'Manage Targets',
```

Comment:
we're not managing targets in TP anymore?

Reply:
no, i just reshuffled the menu a bit